packages/css/src/components/button-row/button-row.scss

```diff
@@ -1,4 +1,5 @@
 .dso-button-row {
+  align-items: center;
```

Comment:
Heb je in Percy gezien dat er iets anders is?

![image](https://user-images.githubusercontent.com/9696261/188443545-0564d4b8-b054-4f03-bbe6-6fef57902f0a.png)

Reply:
Ik vond de oude uitlijning beter, maar laten we het maar even voor wat het is.